index.html

```diff
@@ -395,6 +395,28 @@ <h2>Convert a <var>name</var> to a <var>timestamp</var></h2>
         The <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface was defined in [[NAVIGATION-TIMING]] and is now considered obsolete. The use of names from the <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface is supported to remain backwards compatible, but there are no plans to extend this functionality to names in the <a data-cite="NAVIGATION-TIMING-2#dom-performancenavigationtiming">PerformanceNavigationTiming</a> interface defined in [[NAVIGATION-TIMING-2]] (or other interfaces) in the future.
       </p>
     </section>
+  </section><section id="recommended-mark-names" class="informative">
+    <h2>Recommended mark names</h2>
+    <p>Developers are encouraged to use the following recommended mark names to
+    mark common timings. The user agent does not validate that the usage of
+    the recommended mark name is appropriate or consistent with its
+    description.</p>
+    <p class="note">Adding such recommended mark names can help performance
+    tools tailor guidance to a site. These mark names can also help real user
+    monitoring providers and browsers collect web developer signals regarding
+    their application's performance at scale, and surface this information to
+    developers without requiring any site-specific work.</p>
+    <dl>
+      <dt>"mark_fully_loaded"</dt>
```

Comment:
These seem definitions to me.

Reply:
Yeah. We should probably use [<dfn>](https://respec.org/docs/#definitions-and-linking) to define these.

Reply:
I think GitHub ate an HTML tag here, @yoavweiss. What should we use to define these?